assembl/utils.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think the methods in this file should be placed elsewhere. Not sure yet where `format_date` should go, but the rest are model fetches. So, perhaps make a `assembl/models/utils.py` and place those there. @maparent was there a reason why we never had a global Assembl util module?